src/java/main/org/apache/zookeeper/common/X509UtilTest.java

```diff
@@ -0,0 +1,231 @@
+/**
```

Comment:
This is supposed to be a test file, right? If so, test file should be under src/test folder. Does this even build?